src/@context/Asset.tsx

```diff
@@ -58,110 +59,125 @@ function AssetProvider({
 
   const newCancelToken = useCancelToken()
 
-  const fetchDdo = async (token?: CancelToken) => {
-    LoggerInstance.log('[asset] Init asset, get DDO')
-    setLoading(true)
-    const ddo = await retrieveDDO(did, token)
-
-    if (!ddo) {
-      setError(
-        `[asset] The DDO for ${did} was not found in MetadataCache. If you just published a new data set, wait some seconds and refresh this page.`
-      )
-    } else {
-      setError(undefined)
-    }
-    setLoading(false)
-    return ddo
-  }
+  const fetchAsset = useCallback(
+    async (token?: CancelToken) => {
+      LoggerInstance.log('[asset] Init asset, get assetExtended')
+      setLoading(true)
+      const assetExtended = await retrieveDDO(did, token)
```

Comment:
slightly confusing naming in this method as this is getting `Asset`, but not `AssetExtended`

Reply:
There are probably still more ddos left, just trying to get this out of the way to fix the consume.